### DISCUSSION TITLE
Lesson 3: Fund Me

### DISCUSSION THREAD

**Discussion by formulaRoot** [smartcontractkit/full-blockchain-solidity-course-py#1414] [General]
If you are unable to connect via injected web3 and you are getting the following:

`Currently you have no contract instances to interact with.` 

The fix is simple, regardless of the browser that you are using. 

The issue is your meta mask wallet is not connect to remix. 
To connect it you simply have to be on the remix webpage and then open your meta mask and do the following:

1. click on the 3 dots next to your wallet 
2. click on connected sites 
3. click on remix.

thats it. you will be connected. 

Yes, it might still not work after that, I know. 

1. Save with CMD + S on all your files
2. Refresh and then make sure that you are still connected
3. open your files and go to the Deploy tab 
4. Select Injected Web3

**Comment by cromewar** [comment]
Thanks for the tip @formulaRoot ;)

**Reply by Royalaviation18** [reply]
Thanks, man, it really did help @formulaRoot  :)
